lib/rock/webapp/tasks/api.rb

```diff
@@ -230,6 +230,21 @@ def get_port(name_service, name, port_name, init = false, timeout = 30)
                         end
                         
                     end
+                    
+                    get ':name_service/:name/ports/:port_name/connect' do
+                        puts "connecting to #{request.params["to"]}"
```

Comment:
forgot to remove the output (the return value is transmitted, that should be enough feedback)